src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -97,7 +98,7 @@ def _complex_params(command_group):
         with self.argument_context('{} server restore'. format(command_group)) as c:
             c.argument('server_name', options_list=['--name', '-n'], arg_type=overriding_none_arg_type)
             c.argument('source_server', options_list=['--source-server', '-s'], help='The name or resource ID of the source server to restore from.')
-            c.argument('restore_point_in_time', options_list=['--restore-point-in-time', '--pitr-time'], help='The point in time to restore from (ISO8601 format), e.g., 2017-04-26T02:10:00+08:00')
+            c.argument('restore_point_in_time', options_list=['--restore-point-in-time', '--pitr-time'], help='The point in time in UTC to restore from (ISO8601 format), e.g., 2017-04-26T02:10:00+08:00')
```

Comment:
Why no `default` and no `zone` here? It looks different from `'{} flexible-server restore'.format(command_group)`

Reply:
The PG flexible server team reached out to me to add the zone parameter for flexible server to align it with the portal experience.